src/index.html

```diff
@@ -11,12 +11,97 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of company MOYO"
+          class="moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item name-1">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item name-2">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item name-3">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item name-4">
+            <a
+              href="#laptopsandcomputers"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
```

Comment:
Add attributes to fix this tests
<img width="1512" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/f826e900-8f19-4a41-b530-b48b69229b73">
```suggestion
            <a
              href="#laptopsandcomputers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```